Src/FluentAssertions/Xml/Equivalency/XmlReaderValidator.cs

```diff
@@ -209,5 +226,27 @@ private Failure ValidateText()
 
             return null;
         }
+
+        private Failure ValidateAgainstNulls()
+        {
+            if ((expectationReader is null) ^ (subjectReader is null))
+            {
+                return new Failure(
+                    "Expected {context:subject} to be equivalent to {0}{reason}, but found {1}.",
+                    GetReaderContent(subjectReader), GetReaderContent(expectationReader));
+            }
+
+            return null;
+
+            static string GetReaderContent(XmlReader reader)
```

Comment:
🤔 Isn't it cleaner to create a little `IValueFormatter` for XML readers?

Reply:
Ah, didn't think of that